### PR TITLE
[MAIN] add new brand

### PR DESCRIPTION
**Pull request by apobletts**
* logo
* header colors

### CODE REVIEW

File: 01-Login/templates/unauthorized.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
       <a href="https://passage.id/"><div class="passageLogo"></div></a>
       <div class="header-text">Passage Go Example App</div>
       <div class="spacer"></div>
-      <a href="https://passage.id/" class="link" style="color: white">Go to Passage</span></a>
+      <a href="https://passage.id/" class="header-link">Go to Passage</span></a>
```
Comment (calhorn95) [inline 01-Login/templates/unauthorized.html]: is there supposed to be a span in here

File: 01-Login/templates/index.html [hunk 2]
```diff
@@ -39,7 +39,7 @@
       <a href="https://passage.id/"><div class="passage-logo"></div></a>
       <div class="header-text">Passage Go Example App</div>
       <div class="spacer"></div>
-      <a href="https://passage.id/"><span class="text">Go to Passage</span></a>
+      <a href="https://passage.id/" class="header-link">Go to Passage</span></a>
```
Comment (calhorn95) [inline 01-Login/templates/index.html:42]: did you mean to delete the opening of the `<span>`

**Comment by calhorn95** [comment]
Can you update the logo in the README as well

File: 01-Login/templates/assets/style.css [hunk 3]
```diff
@@ -2,16 +2,22 @@
...
 }
+
+.header-link {
+  color: white;
+  text-decoration-color: white;
+}
   
 .passage-logo {
-  background-image: url('https://storage.googleapis.com/passage-docs/passage-logo.svg');
+  background-image: url('https://storage.googleapis.com/passage-docs/passage-logo-dark.svg');
```
Comment (dylanbrookes) [inline 01-Login/templates/assets/style.css:20]: Is this logo and the other background-image logo that was changed meant to be just the logo icon or should it include the 'Passage' text next to it like in https://storage.googleapis.com/passage-docs/passage-logo-dark-full.png?

**Review by dylanbrookes — APPROVED** [review]

**Review by flanagankp — APPROVED** [review]

File: 01-Login/templates/assets/style.css (hunk 3 above)
Comment (flanagankp) [reply 01-Login/templates/assets/style.css:20]: yes we just use the logo icon without the Passage text in the examples

**Pull request merged**